packages/analytics/amplify_analytics_pinpoint/.metadata

```diff
@@ -0,0 +1,45 @@
+# This file tracks properties of this Flutter project.
+# Used by Flutter tool to assess capabilities and perform upgrades etc.
+#
+# This file should be version controlled.
+
+version:
+  revision: 52b3dc25f6471c27b2144594abb11c741cb88f57
+  channel: stable
+
+project_type: app
```

Comment:
Thanks good catch there.  It appears this was created on the flutter create . command not sure how/why it was modified since then though.  It should be project_type plugin so I changed it.  Looks to be used by flutter when performing upgrades so making this change should be fine. 

Used this as a resource in case you're curious to learn more about what metadata is: https://dev.to/swimmingkiim/what-is-metadata-file-in-flutter-project-38m3